src/app/tabs/analysis-tab.js

```diff
@@ -1,13 +1,12 @@
 var yo = require('yo-yo')
 var css = require('./styles/analysis-tab-styles')
 
-function analysisTab (container) {
+function analysisTab ({ api: {}, events: {} } = {}) {
   var el = yo`
     <div class="${css.analysisTabView} "id="staticanalysisView">
     </div>
   `
-  container.appendChild(el)
-  return el
+  return { render () { return el } }
```

Comment:
can we add `render` as a proper function of `analysisTab` here?

Reply:
yes, i will make a few more PRs to bring tab modules in shape

Reply:
ok